.idea/vcs.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+-->
+<project version="4">
```

Comment:
My IDEA regenerates this file from time to time, which is a bit annoying as I need to exclude it from the commit...

Does anyone else hit the same issue? I'm thinking maybe it's better to put this in the developer guide, not in the config file. Then people can do it by themselves, maybe even with UI instead of modifying the raw config files directly.

Reply:
cc @HyukjinKwon @dongjoon-hyun @viirya

Reply:
I don't know exactly how, probably because I switch projects frequently on IDEA. At least this happens when I re-import the Spark repo as a sbt project in IDEA.